cms/jinja2/templates/pages/release_calendar/release_calendar_page--confirmed.html

```diff
@@ -3,8 +3,35 @@
 
 {% block header_area %}
     {%- set hero_topic = _("Release") -%}
-    {%- set release_date_label = _("Release date:") -%}
     {%- set breadcrumbs_label = _("Breadcrumbs") -%}
+    {%- set release_date_label = _("Release date:") -%}
+    {%- set next_release_date_label = _("Next release date:") -%}
```

Comment:
Back to the original comment: the current live site doesn't show a next release date, nor were the requirements asking for this.

What prompted this change?

Reply:
closing in favour of the discussion in https://github.com/ONSdigital/dis-wagtail/pull/158#discussion_r2063907359